pkg/controller/datavolume/controller-base.go

```diff
@@ -412,19 +412,34 @@ func updatePendingDataVolumesGauge(ctx context.Context, log logr.Logger, dv *cdi
 		return
 	}
 
```

Comment:
I basically agree, but since it's a small PR that is quite easy to review, I think splitting it now is a waste of time, and it will be squashed anyway in merge. However, I'll sure learn this lesson for the next time.

Reply:
I keep forgetting the (redundant) squash we are doing. If it weren't for the squash I might have insisted since its helpful also for the future, but with the squash and the fact I already review it is a waste of time as you mentioned.